BorrowBookControl.java

```diff
@@ -37,39 +37,39 @@ public void memberCardSwiped(int memberId) {
 		}
 		member = library1.getMember(memberId);
 		if (member == null) {
-			borrowBookUi.Display("Invalid memberId");
+			borrowBookUi.displays("Invalid memberId");
 			return;
 		}
 		if (library1.memberCanBorrow(member)) {
 			pendingList = new ArrayList<>();
-			borrowBookUi.setBorrowBookUiState(BorrowBookUI.BorrowBookUiState.SCANNING);
+			borrowBookUi.setBorrowBookUiState(BorrowBookUi.BorrowBookUiState.SCANNING);
 			borrowBookState = BookControlState.SCANNING;
 		} else {
-			borrowBookUi.Display("Member cannot borrow at this time");
-			borrowBookUi.setBorrowBookUiState(BorrowBookUI.BorrowBookUiState.RESTRICTED);
+			borrowBookUi.displays("Member cannot borrow at this time");
+			borrowBookUi.setBorrowBookUiState(BorrowBookUi.BorrowBookUiState.RESTRICTED);
 		}
 	}
 
-	public void scanneBook(int bookId) {
+	public void scannedBook(int bookId) {

```

Comment:
it should be scannbook as its currently spanning it its not past tense

Reply:
Okay Joey I will do it. Thanks